templates/views/index.html

```diff
@@ -120,6 +110,18 @@ <h4>Support our cause</h4>
                 </div>
                 <img src="/images/PreciousCadeton.jpg" class="__shecodetestimimg">
             </div>
+            <div class="col-lg-4 col-md-4 col-sm-4 text-center">
+                <div class="__shecodetestimcard last-child">
+                    <p class="__shecodetestimp">
+                        Through She Code Africa, I have met lots of people that have been helpful in my journey so far.
+                        I am also privileged to
+                        be part of the Cohort 1 of SCA's Mentorship Program and I have learnt so much within a short
+                        period.
+                    </p>
```

Comment:
Please add a line break/break tag before the name so the name is on equal length with the 2 other cards